### ISSUE TITLE
[SUGGESTIONS] G-Colab and better text detection

### ISSUE THREAD

**Issue by forart** [YaoFANGUK/video-subtitle-extractor#11]
Hi there, I didn't tested it yet (since I don't own any discrete GPUs in my main hw configs) but your software seems cool.

It would be great for people like me to have a Google-Colab version of it.

Check out these in order to have an idea:
- https://colab.research.google.com/github/AlphaAtlas/VapourSynthColab/blob/master/VapourSynthColab.ipynb
- https://colab.research.google.com/drive/1arRAJUPBC3a3l2iGghbs3CqDjdD1IQ6y

You can build your own (of course) but you can also ask to add it to MixLab:
- https://colab.research.google.com/github/shirooo39/MiXLab/blob/master/MiXLab.ipynb

Last but not least, it would be great to establish some kind of collaboration with these projects in order to achieve a better text detection:
- https://github.com/shawnsky/extract-subtitles
- https://github.com/Dinghow/OCR_on_Video
- https://github.com/krohak/Embedded-Subtitles-OCR
- https://github.com/PatchyVideo/MMD-Translator

Hope that inspires !

**Comment by YaoFANGUK** [comment]
Thank you for your great suggestions. I would like to refactor the G-Colab version of it. Unlike GUI version, in which users can easily select subtitle area and filter out scene text, it is challenging to think out an algorithm that determines subtitle area in command line interface. Thus, currently the user experience with G-Clobal version is actually not good. I did notice that Colab can have so many features and your suggestion reallty helps me.

**Comment by forart** [comment]
In MixLab, there's aria/transmission/utorrent that have nice an functional GUIs.

Don't you think is possible to build one for your G-Colab notebook ?

If not, can't the local GUI be modded to "interact" with the remote G-Colab notebook ?

**Comment by YaoFANGUK** [comment]
Yeah, it is possible to do that but it can end up costing me a lot of time. I will have a try. Hope can build one soon.
